src/EnergyPlus/WindowManager.cc

```diff
@@ -144,6 +144,11 @@ namespace WindowManager {
 
```

Comment:
No actual code change in this file either. Just refactoring and reformatting by clang.